docs/migration/2_0.md

```diff
@@ -76,3 +77,5 @@ Previously config validator logic was part of `ConfigModule`. If you are not usi
 - `OrderDetailItemsComponent` now requires `PromotionService`. This service needs to be provided for `OrderDetailItemsComponent`.
 - `OrderConfirmationItemsComponent` now requires `PromotionService`. This service needs to be provided for `OrderConfirmationItemsComponent`.
 - `CartItemComponent` now requires `PromotionService`. This service needs to be provided for `CartItemComponent`.
+- `CartCouponComponent` no longer uses `CartService`. This service usage was replaced with corresponding methods from `ActiveCartService`. Also `CustomerCouponService` and `FeatureConfigService` are now required parameters. These services needs to be provided for `CartCouponComponent`.
```

Comment:
Hopefully `FeatureConfigService` will got removed before 2.0 from every class constructors. We have 2 options:
a) not menotion it here in the docs now
b) mention now, but remember to remove it later (note in a relevant ticket).
It's up to you.

(it applies also to the next line)

Reply:
I choose 'b' because I always want to keep `develop` in a releasable state. No matter if we will be able to remove all deprecations.